Makefile

```diff
@@ -60,6 +60,6 @@ setup:
 test:
 	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive --timeout 10000
 	$(ISTANBUL) check-coverage --branches 100
-ifneq ($(shell node --version | sed 's/[.][^.]*$$//'),v0.10)
+ifeq ($(shell node --version | cut -d . -f 1),v6)
```

Comment:
After some trial and error, I think I have a version which will do the right thing in most environments:

1.  If `$(TERM)` is `dumb`, <code>$(<a href="https://www.gnu.org/software/make/manual/html_node/Text-Functions.html">patsubst</a> dumb,,$(TERM))</code> will evaluate to empty string; otherwise it will evaluate to the value of `$(TERM)` (which may be empty string).

2.  <code>$(<a href="https://www.gnu.org/software/make/manual/html_node/Conditional-Functions.html#Conditional-Functions">if</a> $(patsubst dumb,,$(TERM)),\033[33m\033[7mWARN\033[27m,[WARN])</code> will evaluate to `[WARN]` if the value of `$(TERM)` is `dumb` (or empty string); `\033[33m\033[7mWARN\033[27m` otherwise. `33m` sets the foreground colour to yellow, `7m` switches foreground and background colours, and `27m` undoes the switch.

3.  This appears inside a `$''` string, but since `$` has special meaning in a makefile we must escape it with an extra dollar sign.

@rjmk, would you mind sanity checking the line below? There's no doubt we're overengineering this, but I believe the only cost is the time I've spent fiddling (which I've quite enjoyed).

One can test this by commenting out lines 61 and 62 then running `make test` and `TERM=dumb make test`. CircleCI is happy now too.